specs/spec_reader.go

```diff
@@ -32,6 +34,22 @@ func expandFileConfig(cfg []byte) ([]byte, error) {
 			expandErr = err
 			return nil
 		}
+		var isJSON any
+		if err := json.Unmarshal(content, &isJSON); err == nil {
+			k := reflect.TypeOf(isJSON).Kind()
+			if k == reflect.Map || k == reflect.Slice {
+				buffer := &bytes.Buffer{}
+				encoder := json.NewEncoder(buffer)
+				encoder.SetEscapeHTML(false)
+				expandErr = encoder.Encode(string(content))
+				b := buffer.Bytes()
+				if l := len(b); l > 0 && b[l-1] == '\n' {
```

Comment:
`encoder.Encode` always adds the trailing `\n`, there's no way to unset `EscapeHTML` otherwise.